packages/calcite-components/src/components/navigation-user/navigation-user.e2e.ts

```diff
@@ -22,7 +22,7 @@ describe("calcite-navigation-user", () => {
       },
       {
         propertyName: "textDisabled",
-        value: "",
+        value: true,
```

Comment:
In Stencil, casting is done both to property and attribute (thus setting boolean property to `""` will cast to `true`). In Lit, casting is only done when converting attribute to property or property to attribute.

See https://qawebgis.esri.com/arcgis-components/?path=/docs/references-lumina-transition-from-stencil--docs#attribute-to-property-casting